src/Microsoft.ML.Api/TypedCursor.cs

```diff
@@ -26,6 +26,21 @@ public interface IRow<TRow> : IRow
         void FillValues(TRow row);
     }
 
+    /// <summary>
+    /// This interface is an <see cref="IRow"/> with 'stringly typed' binding.
```

Comment:
>stringly [](start = 54, length = 8)

I like the word, but you probably meant 'strongly' ?

Reply:
I should have just copy-pasted. :D

---
In reply to: [219578585](https://github.com/dotnet/machinelearning/pull/973#discussion_r219578585) [](ancestors = 219578585)